spec/commands/order/all_food_orders_spec.rb

```diff
@@ -8,12 +9,34 @@
     expect(all_food_orders.applies_to("all food orders")).to eq(true)
   end
 
-  it "it return the total of the different food " do
+  it "return the total of the different food" do
     Helper.order({user_id: "asdf", user_name: "will", user_message: "burger"})
     Helper.order({user_id: "qwer", user_name: "fabien", user_message: "burger"})
     Helper.order({user_id: "poiu", user_name: "james", user_message: "fish"})
 
     all_food = "burger: 2\nfish: 1"
     expect(all_food_orders.run).to eq(all_food)
   end
+
+  it "return only the meal of the current week" do
```

Comment:
That's something I've noticed in a couple of spec descriptions here but haven't comment on separately. It should read `it "returns ..." do`, `it "sets ..." do`. The "s" is needed in that case (something with third person, singular present tense - I don't really know the actual grammar rule).